docs/tutorials/intro_tutorial.rst

```diff
@@ -990,7 +1004,32 @@ iteration).
 
     245it [00:25,  9.75it/s]
 
+Note: If you are running a Windows OS you may have an issue with `batch_run`. One way to overcome this is to take your
```

Comment:
You should say what the error message on Windows is, so that people know it is the same problem.

Reply:
Also, I don't think it needs to be outside Jupyter. The `freeze_support()` can be run inside Jupyter.

Reply:
Can't test because I don't have access to a Windows machine.

Reply:
I added a note before the cell. If  you change number of processes to 1 it works on WIndows OS. Even adding freeze_support etc you get the same error. Based on the traceback Jupyter is blocking multiprocessing's ability to get the task.